arch/x86/Kconfig

```diff
@@ -90,12 +90,19 @@ config X86_NO_MELTDOWN
 	  This hidden option should be set on a per-SOC basis to indicate that
 	  a particular SOC is not vulnerable to the Meltdown CPU vulnerability.
 
-config X86_NO_SPECTRE
+config X86_NO_SPECTRE_V2
 	bool
 	default y
 	help
 	  This hidden option should be set on a per-SOC basis to indicate that
 	  a particular SOC is not vulnerable to the Spectre V2 CPU vulnerability.
+
+config X86_NO_SPECTRE_V4
+	bool
+	default y
+	help
+	  This hidden option should be set on a per-SOC basis to indicate that
+	  a particular SOC is not vulnerable to the Spectre V4 CPU vulnerability.
```

Comment:
Should set this on the existing non-qemu targets, otherwise we've added useless code.

Reply:
It's the same for CONFIG_NO_MELTDOWN and CONFIG_NO_SPECTRE_V2.  It's there for SoC/boards targets to set; none of the ones that are available publicly need that as far as I can tell.

Reply:
Just checked again, and Minnowboards seem to be affected (they use Series E Atom processors).  Added a commit to the PR setting this to `n` for that board.